app/resources/api/v2/request_resource.rb

```diff
@@ -21,27 +77,90 @@ class RequestResource < BaseResource
+      #   @return [String] The unique identifier for the request.
+      #   @note This identifier is automatically assigned upon creation and cannot be modified.
       attribute :uuid, readonly: true
+
+      # @!attribute [rw] role
+      #   The role of the request, such as "analysis" or "testing."
+      #   @return [String] The role of the request.
+      #   @note This attribute can only be set once.
       attribute :role, write_once: true
+
+      # @!attribute [rw] state
+      #   The current state of the request, such as "pending," "completed," or "in progress."
+      #   @return [String] The state of the request.
+      #   @note This field can be updated during the lifecycle of the request.
       attribute :state
+
+      # @!attribute [rw] priority
+      #   The priority of the request, such as "high," "medium," or "low."
+      #   @return [String] The priority of the request.
+      #   @note This attribute can only be set once.
       attribute :priority, write_once: true
+
+      # @!attribute [r] options
+      #   Additional options related to the request, such as configurations or metadata.
+      #   @return [Hash] A hash containing additional options for the request.
+      #   @note This attribute is read-only.
       attribute :options, readonly: true
+
+      # @!attribute [r] library_type
+      #   The type of library associated with the request, e.g., "scRNA" or "DNA."
+      #   @return [String] The type of library associated with the request.
+      #   @note This attribute is read-only.
       attribute :library_type, readonly: true
 
+      ###
       # Relationships
+      ###
+
+      # @note `always_include_linkage_data` is currently broken. It is supposed to ensure that
+      #   linkage data is always included in the response. However, it is not working as expected.
+      #   To ensure linkage data is always included, explicitly include the relationship in the request.
+      #   For example, use `?include=submission` in the request URL to include the `submission` relationship.
+      #   See Y25-164 for more details and a possible fix.
+      # @!attribute [r] submission
+      #   The submission associated with this request.
       has_one :submission, always_include_linkage_data: true
+
+      # @note `always_include_linkage_data` is currently broken. It is supposed to ensure that
+      #   linkage data is always included in the response. However, it is not working as expected.
+      #   To ensure linkage data is always included, explicitly include the relationship in the request.
+      #   For example, use `?include=order` in the request URL to include the `submission` relationship.
+      #   See Y25-164 for more details and a possible fix.
+      # @!attribute [r] order
+      #   The order associated with this request.
       has_one :order, always_include_linkage_data: true
-      # NB. adding always_include_linkage_data: true caused a problem in the JSON Api client in Limber in that it has
```

Comment:
It seems inconsistent to have `always_include_linkage_data: true` on the submission and order on the right, with comments saying it doesn't work, but then not have it on this request_type (and also remove the comment saying why it's not there).
If it's not working (as the comments on the right say), then maybe it shouldn't be present on submission and order, and maybe all 3 fields should have comments saying it was removed and what to do instead?